### ISSUE TITLE
how to disable P3 colors when generating css using @cobalt-ui/plugin-css

### ISSUE THREAD

**Issue by uptonking** [drwpow/cobalt-ui#23]
- P3 colors is good for production
- but during development, p3 color values is not as explict as hex values
- 🤔 is there an option to disable it 

> I've read docs in [@cobalt-ui/plugin-css
](https://cobalt-ui.pages.dev/docs/plugins/css/)

**Comment by drwpow** [comment]
This is a good suggestion. We should add a `p3` option (or something) in the config to disable P3 colors if desired (but generate them by default).

This plugin doesn’t distinguish between “dev” and “prod” but if you wanted to toggle it on or off based on env you could use `process.env.NODE_ENV`, e.g.:

```ts
import pluginCSS from '@cobalt-ui/plugin-css';

/** @type import('@cobalt-ui/core').Config */
export default {
  plugins: [
    pluginCSS({
      p3: process.env.NODE_ENV === 'production'
    }),
  ],
}
```

**Comment by drwpow** [comment]
Released in `v1.3.0`! 🎉 Just set `p3: false` in the plugin options to disable it.
